documentation/asciidoc/accessories/camera/libcamera_apps_getting_help.adoc

```diff
@@ -0,0 +1,19 @@
+=== Getting Help
+
+For further help with `libcamera` and the `libcamera-apps`, the first port of call will usually be the https://www.raspberrypi.org/forums/viewforum.php?f=43[Rasperry Pi Camera Forum]. Before posting, it's helpful to:
+
+* Ensure your https://www.raspberrypi.org/documentation/computers/os.html#using-apt[software is up to date].
```

Comment:
Can you make this a relative-link please?

Reply:
Sure... what would a relative link look like?  :)

Reply:
`xref:os.adoc#using-apt[software is up to date]`
(which I just stole from https://github.com/raspberrypi/documentation/blob/develop/documentation/asciidoc/computers/using_linux/using-the-terminal.adoc )

Reply:
I think that's done now. I had to use `xref:../computers/os.adoc#using-apt[blah blah]` though!